frontend/ui/widget/menu.lua

```diff
@@ -666,12 +671,16 @@ function Menu:init()
         title_face = self.title_face,
         title_multilines = self.title_multilines,
         title_shrink_font_to_fit = self.title_shrink_font_to_fit,
-        subtitle = self.show_path and BD.directory(filemanagerutil.abbreviate(self.path)),
+        subtitle = self.title_bar_fm_style and "" or (self.show_path and BD.directory(filemanagerutil.abbreviate(self.path))),
```

Comment:
This seems to mean that if we provide a `title_bar_fm_style=true` and `subtitle="something"`, that "something" won't be shown, and one has to use switchItemTable or some other method to actually set it?

Reply:
Or just add this above there still in Menu:init() ?:
```lua
if self.title_bar_fm_style and not self.subtitle then
    self.subtitle = "" -- we need a subtitle
end
```